### DISCUSSION TITLE
Selecting palettes in character selection

### DISCUSSION THREAD

**Discussion by sfz3mix** [ikemen-engine/Ikemen-GO#914] [Feature Request] [answered]
*Preview palette before selecting the character

Suggestion:
In character selection, if you have an animated portrait when holding start press the directional does not change the character, but rather the character's palette, leaving the player to experiment with the palettes in the character selection itself.  

I have tried to describe the simplest way to do it but any means of palette selection in the selection screen will be well received.

**Comment by Paul45mec** [comment]
surely the static big portrait is not possible, but the small animated portraits based on stance 0 it can be totally possible, even you need to create a new stance including the palettes for buttons attributed to that.

**Comment by K4thos** [comment]
This feature will be implemented, it's only a matter of time. The code for it is ready but it depends on unstable branch of the engine with refactored lua scripts, so it won't be available in the next build that will be published soon. Right now the plan is to make it available on 0.100.

**Reply by Paul45mec** [reply]
That's an amazing news, it's a satisfaction to see this engine evolving everyday more and adding classic games features so cool like that. the Devs are doing a fantastic work, imagine how will be nice you can choose the char colors on select screen, but will it be possible to choose alternate mode too? 

Example : Ryu and Evil Ryu... 6 colors for Ryu and 6 colors for Evil Ryu.

**Comment by K4thos** [comment, marked as answer]
already implemented in nightly
